buttons/button_base.h

```diff
@@ -81,6 +97,8 @@ class ButtonBase : public Looper,
     STDOUT.println(" button");
   }
 
+  bool doubleclicked_ = false;
+  bool doubleclickedandheld_ = false;
   const char* name_;
```

Comment:
doubleclicked_ is used to flag that a double click has started
doubleclickedandheld_ is used to flag that the second click was held down long enough to be a HELD event, to ensure that DOUBLE_CLICK doesn't fire after a DOUBLE_CLICK_AND_HELD